Bottom: 8cb34bf6d9e038b3c04f42637d7d19ccbd6478d0
Top:    15be28088889530c7ec38102f6d6b070ff4da9ad
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-18 22:41:07 +0000

Add warning about bogus code in strflags.c


---

diff --git a/src/strflags.c b/src/strflags.c
index 6cb6548..e6ce175 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -575,6 +575,7 @@ dump_flag (FlagType * f)
 {
   int l;
   printf ("F:%08x T:[", f->f);
+#warning THIS NEXT LINE IS BOGUS, THE ARRAY IS HALF THIS LENGTH
   for (l = 0; l < (MAX_LAYER + 7) / 8; l++)
     printf (" %02x", f->t[l]);
   printf ("]");
